packages/magento-wishlist/components/WishlistItemsActionCards/WishlistItemsActionCards.tsx

```diff
@@ -0,0 +1,16 @@
+import { useWishlistItems } from '../../hooks'
+import { WishlistItemActionCard } from '../WishlistItemActionCard/WishlistItemActionCard'
+
+export function WishlistItemsActionCards() {
+  const wishlistItemsData = useWishlistItems()
+
+  /** Structure between guest and customer wishlist differs */
+  return (
+    <>
+      {wishlistItemsData.data?.map((item) => {
+        if (!item?.uid && !item?.id) return null
+        return <WishlistItemActionCard variant='default' {...item} />
+      })}
+    </>
+  )
+}
```

Comment:
This component doesn't make sense to me. What is the function of this component? Where can I see the difference between the structures?